include/core/JamomaSample.h

```diff
@@ -44,8 +44,9 @@ namespace Jamoma {
 	class SampleBundle {
 		using SampleBundleData = std::vector<SampleVector>;
 		
-		SampleBundleData	mChannels;		// each item in this vector represents a channel, which itself is a vector of samples
-		size_t				mFrameCount;	// if we change the number of channels we could loose the framecount in the vectors contained by mChannels, so we cache it here.
+		SampleBundleData	mChannels;		//!< each item in this vector represents a channel, which itself is a vector of samples
```

Comment:
I've used `///<` for Doxygen comments on the tail of a line before, but not `//!<`.  Is there any benefit or difference with this alternate syntax @nwolek ?